tools/lsp/common.rs

```diff
@@ -677,7 +677,7 @@ pub enum PreviewToLspMessage {
     /// Show a status message in the editor
     Status { message: String, health: crate::lsp_ext::Health },
     /// Report diagnostics to editor.
-    Diagnostics { uri: Url, diagnostics: Vec<lsp_types::Diagnostic> },
+    Diagnostics { uri: Url, version: UrlVersion, diagnostics: Vec<lsp_types::Diagnostic> },
```

Comment:
btw, what's the difference between `SourceFileVersion` ans `UrlVersion`? Should we not get rid of `UrlVersion` in favor of `SourceFileVersion` everywhere?

Reply:
That is already merged as part of #5842.